### DISCUSSION TITLE
voice2RVC

### DISCUSSION THREAD

**Discussion by kalle07** [erew123/alltalk_tts#470] [Q&A] [answered]
if iam here 
![grafik](https://github.com/user-attachments/assets/d9c1e556-97f5-4ec0-a646-f41e94a2e5b3)

is it possible or useful to set the language ... can this improve the output or does it have nothing to do with it?

**Comment by erew123** [comment, marked as answer]
RVC is a Pitch changer and not a text to speech engine. As such it doesnt support language it just manipulates audio. There is no language option to set.
